### ISSUE TITLE
28738194: UISegmentedControl in UITableView section header has wrong UIAccessibilityTraits

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16103]
#### Description

Summary:
When a UISegmentedControl is placed in a UITableView section header it uses incorrect accessibility traits.

Steps to Reproduce:
Run the attached sample app and enable VoiceOver (or the Accessibility Inspector)

Expected Results:
When hovering over the segmented control, it should read "First, Button, Selected, 1 of 2".
So the segmented control item should have the traits: UIAccessibilityTraitButton, UIAccessibilityTraitSelected, plus the private trait for the segmented control ("1 of 2").

Actual Results:
It reads "First, Header". The segmented  control item has the traits: UIAccessibilityTraitStaticText, UIAccessibilityTraitHeader.

Note:
This affects us badly in our Where To? app (http://itunes.apple.com/app/id903955898?mt=8) because the user doesn't understand that the segmented control in the details view is actually interactive and not static text.
Also there's no workaround (other than completely rewriting UISegmentedControl) because the UISegment subview is not public and even if it where, its accessibilityTraits property is not writable.
## 

Product Version: 10.1 beta 3
Created: 2016-10-12T18:12:36.568780
Originated: 2016-10-12T20:12:00
Open Radar Link: http://www.openradar.me/28738194
